items.yaml

```
@@ -15,4 +15,16 @@ Axe:
   type: "Weapon"
   damage: 7
   defend: 5
-  description: "An ancient axe found in the dungeons. You wonder what happened to it's previous owner..."
\ No newline at end of file
+  description: "An ancient axe found in the dungeons. You wonder what happened to it's previous owner..."
+Leather Jacket:
+  type: "Armor Piece"
+  armor protection: .6
+  description: "A small jacket that may not protect much but keeps you warm when it's cold."
+Leather Boots:
+  type: "Armor Piece"
+  armor protection: .1
+  description: "These boots may be dirt or wet but they let you have great mobility."
```

```suggestion
  description: "While these boots are dirty and wet, they grant you great mobility."
```

I presume they also get a great `swiftness` value.

---

Maybe. I'll make an other PR for the swiftness attribute I think.